oauth2-server-snap/lib/Network/OAuth2/Server/Snap.hs

```diff
@@ -64,6 +104,15 @@ createGrant
     :: Handler b (OAuth2 IO b) TokenGrant
 createGrant = return aGrant
 
+-- | Create an access token and send it to the client.
+createAndServeToken
```

Comment:
Should probably be called `clientGrant`. Would be good to have a link to the RFC section which describes it too.